kong/globalpatches.lua

```diff
@@ -467,6 +467,9 @@ return function(options)
       -- to "all".
       local instrumentation = require "kong.tracing.instrumentation"
       instrumentation.set_patch_dns_query_fn(toip, function(wrap)
+        -- TODO: this call has no side effect.
+        -- maybe client.toip = wrap?
+        -- but that doesn't work to client = require("kong.tools.dns")() case
```

Comment:
```suggestion
```

It has a side effect. The function exported uses this local variable as its up value.